packages/obuilder/obuilder.0.4/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis: "Run build scripts for CI"
+description:
+  "OBuilder takes a build script (similar to a Dockerfile) and performs the steps in it in a sandboxed environment."
+maintainer: ["talex5@gmail.com"]
+authors: ["talex5@gmail.com"]
+homepage: "https://github.com/ocurrent/obuilder"
+doc: "https://ocurrent.github.io/obuilder/"
+bug-reports: "https://github.com/ocurrent/obuilder/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "lwt" {>= "5.5.0"}
+  "astring"
+  "fmt" {>= "0.8.9"}
+  "logs"
+  "cmdliner" {>= "1.1.0"}
+  "tar-unix" {>= "2.0.0"}
+  "yojson"
+  "sexplib"
+  "ppx_deriving"
+  "ppx_sexp_conv"
+  "sha"
+  "sqlite3"
+  "obuilder-spec" {= version}
+  "ocaml" {>= "4.10.0"}
+  "alcotest-lwt" {with-test}
+  "odoc" {with-doc}
+]
+build: [
```

Comment:
```suggestion
conflicts: [ "result" {< "1.5"} ]
build: [
```
For  [(failed: Unbound value Result.bind)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/7fe26856e905c172d537f3d35a9390755d8a9fbd/variant/opam-2.1,compilers,4.13,obuilder.0.4,lower-bounds)